surround360_camera_ctl/README.md

```diff
@@ -106,7 +106,7 @@ After installing all of the dependencies as described above, run:
 To test that compilation is successful, run:
 
 ```
-  ./bin/CameraControl -numcams 17 -raw -nbits 8 -shutter 20 -gain 0 -debug
+  sudo ./bin/CameraControl -numcams 17 -raw -nbits 8 -shutter 20 -gain 0 -debug
```

Comment:
Hi @Reality7 ,

If we don't run ```scripts/setup.sh```,  normally we will not have directory ```/media/snoraid```. If we want to create this directory automatically when we run ```./bin/CameraControl``` for the first time, we should run it with root privilege. Because  creating directory under ```/media``` will need root privilege.